app/scripts/controllers/split-view.js

```diff
@@ -13,7 +13,9 @@ angular
 
 /* @ngInject */
 function splitViewController(uitidAuth, $scope) {
+  const urlParams = new URLSearchParams(window.location.search);
```

Comment:
I might be mistaken, but this no longer seems necessary now that we use `window.top` instead of an url parameter?

Reply:
True, will remove it.